src/chi/utilities/anchor.scss

```diff
@@ -0,0 +1,51 @@
+@import 'mixins';
+@import 'variables';
+
+.chi {
+
+  .-anchor {
+    &::before {
+      content: '';
+      display: block;
+      height: 5rem;
+      margin: -5rem 0 0;
```

Comment:
@davidojedacl for what is it necessary?

Reply:
It is necessary to maintain the distance between divs in documentation
<img width="1154" alt="screen shot 2019-02-19 at 4 17 38 pm" src="https://user-images.githubusercontent.com/43439742/53026166-aae3b580-3462-11e9-8a8d-5f254bf18e00.png">

Reply:
@davidojedacl these properties must be in chi-docs.scss and can not be as part of the general anchor.scss, as we don't want to make it generic